src/extensions/scratch3_video_sensing/index.js

```diff
@@ -359,12 +378,12 @@ class Scratch3VideoSensingBlocks {
                         MOTION_DIRECTION: {
                             type: ArgumentType.NUMBER,
                             menu: 'MOTION_DIRECTION',
-                            defaultValue: 1
+                            defaultValue: Scratch3VideoSensingBlocks.MOTION
```

Comment:
😄

Reply:
I followed up in a new PR with better enums!

Reply:
https://github.com/LLK/scratch-vm/pull/1028